src/contexts/FormContext.tsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+import React, { createContext, useContext } from 'react'
+import { useFormik } from 'formik'
+import { validate } from '../validation/FormValidation'
+
+const initialValues = {
+  fullName: '',
+  contactEmail: '',
+  linkedin: '',
+  bio: '',
+  country: '',
+  city: '',
+  openToRelocationCountry: false,
+  openToRelocationCity: false,
+  remoteOnly: false,
+  position: [],
+  seniority: [],
+  employment: [],
+  techStack: '',
+}
+
+const FormContext = createContext<ReturnType<typeof useFormik> | null>(null)
+
+export const FormProvider: React.FC<{ children: React.ReactNode }> = ({
```

Comment:
**issue:** But why? Formik has it out of the box: https://formik.org/docs/api/useFormikContext